CHANGES.MD

```diff
@@ -0,0 +1,19 @@
+
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
Instead of "Unreleased" you could just make this `0.2.0` and create a release after we merge it, so you don't have to go back and fix it after releasing.

Reply:
I'd rather whoever does the release update it after a release is made. Also looking forward jazzband has a great release process https://jazzband.co/about/releases